### PR TITLE
Add applicant class and related classes

### LINKED ISSUE #39: Add Applicant class for job applicants (i.e. modifying Person)
This is basically edit Person class

### PR DESCRIPTION
**Pull request by darren2pro**
Resolves #39

### CODE REVIEW

File: src/main/java/seedu/address/model/applicant/Address.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.applicant;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's address in the address book.
```
Comment (Harry-Gao-H) [inline src/main/java/seedu/address/model/applicant/Address.java:7]: Represents a Applicant's address in the MTR system.

File: src/main/java/seedu/address/model/applicant/Applicant.java [hunk 2]
```diff
@@ -1,3 +1,112 @@
...
+public class Applicant {
+
+    // Identity fields
+    private final Name name;
+    private final Phone phone;
+    private final Email email;
+
+    // Data fields
+    private final Address address;
+    private final Position position;
```
Comment (Harry-Gao-H) [inline src/main/java/seedu/address/model/applicant/Applicant.java:22]: Maybe it is better to put position inIdentity fields?

File: src/main/java/seedu/address/model/applicant/Address.java (hunk 1 above)
Comment (BenjaminTan99) [reply src/main/java/seedu/address/model/applicant/Address.java:7]:
```suggestion
 * Represents an Applicant's address in the MTR system.
```

**Review by BenjaminTan99 — APPROVED** [review]
LGTM.

**Pull request merged**